README.md

```diff
@@ -21,14 +21,16 @@ Currently, the platform offers developers the option to build HIP and ROCm from
 ### Ubuntu
 
 ```bash
+# Install Ubuntu dependencies
+sudo apt install gfortran git git-lfs ninja-build cmake g++ pkg-config xxd patchelf automake python3-venv python3-dev libegl1-mesa-dev
```

Comment:
```suggestion
sudo apt install gfortran git-lfs ninja-build cmake g++ pkg-config xxd patchelf automake python3-venv python3-dev libegl1-mesa-dev
```
[`git-lfs`](https://packages.ubuntu.com/noble/git-lfs) already pulls in `git` thus we can add it to the list but could also omit if we want to keep the list shorter.